cyral/resource_cyral_repository.go

```diff
@@ -16,8 +16,41 @@ import (
 const (
 	mongodbRepoType             = "mongodb"
 	mongodbReplicaSetServerType = "replicaset"
+	repositoryTypesMarkdown     = "\n  - `bigquery`" +
```

Comment:
This should be created out of `repositoryTypes` to avoid type duplication.

Reply:
Thanks Wilson!  Fixed.